test/module/iroha-cli/client_test.cpp

```diff
@@ -292,19 +295,18 @@ TEST_F(ClientServerTest, SendTxWhenStatefulInvalid) {
   ASSERT_EQ(client.sendTx(tx).answer, iroha_cli::CliClient::OK);
 
   // fail the tx
+  auto cmd_name = "CommandName";
+  size_t cmd_index = 2;
+  uint32_t error_code = 2;
```

Comment:
Set please that value different from the `cmd_index` one
Test is a bit more precise in that case